stratio-docs/en/modules/ROOT/pages/installation.adoc

```diff
@@ -0,0 +1,527 @@
+= Installation
+
+== Prerequisites
+
+=== EKS
+
+* Roles and policies
++
+For automated provisioning in EKS, it is necessary to execute actions on various AWS services such as ec2, ecr, eks, elasticloadbalancing, etc. While the use or lack of use of these actions will depend on the type of installation, the provider validates that the indicated user has these permissions in order to run normally.
++
+** xref:attachment$stratio-eks-policy.json [Download permanent permissions for EKS].
+** xref:attachment$stratio-eks-eks-temp-policy.json[Download temporary permissions for EKS]
+** xref:attachment$stratio-eks-eks-temp-policy.json[Download temporary permissions for EKS]
+
+* Certified operating systems
+
+To ensure the functionalities supported by the EKS managed _control-plane_, you should use any AMI provided by AWS specifically for this purpose.
+
+The https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html[AMIs optimized for Amazon EKS] are built on the Amazon Linux 2 operating system.
+
+=== GCP
+
+* Permissions
+
+For deployments on Google Cloud Platform, you will require permissions on _compute_ (instances, disks, images, routers, networks, etc.). Similarly to other supported providers, provisioning requires an account with all requested permissions.
++
+xref:attachment$stratio-gcp-permissions.list[Download permissions for GCP].
+
+* Certified operating systems
++
+For GCP environments, use https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi[_image builder_], an official tool to create and distribute images for _Stratio KEOS_.
++
+The currently recommended operating system for this provider is Ubuntu 22.04.
+
+== Proprietary infrastructure
+
+Although one of the advantages of automatic resource creation in provisioning is the great dynamism it provides, for security and compliance reasons, it is often necessary to create certain resources prior to the deployment of _Stratio KEOS_ in the cloud provider.
+
+As it will be shown below, the provider aupports the use of both a VPC and subnets previously created using the networks parameter in the cluster descriptor.
```

Comment:
```suggestion
As it will be shown below, the provider supports the use of both a VPC and subnets previously created using the networks parameter in the cluster descriptor.
```